config/initializers/rotate_log.rb

```diff
@@ -0,0 +1,10 @@
+# Rotate development and test log files when they exceed 5 MB. Otherwise logs
+# will grow unbounded into gigabytes over the life of a project.
+if Rails.env.development? || Rails.env.test?
+  log_file = Rails.root.join("log", "#{Rails.env}.log")
+
+  if log_file.file? && log_file.size > 5_000_000
+    FileUtils.cp(log_file, "#{log_file}.1")
```

Comment:
What if this is the second time this log is rotated? Should we also go to a `.2`?

...although I'm skeptical *any* dev is going to find 5GB of dev/test logs useful.